internal/executor/config/config.go

```diff
@@ -49,15 +70,27 @@ func load() OctaviusExecutorConfig {
 	if err != nil {
 		return OctaviusExecutorConfig{}
 	}
+	JobPodAnnotation, err := GetMapFromJson(fang, "job_pod_annotations")
+	if err != nil {
+		return OctaviusExecutorConfig{}
```

Comment:
return the error so the caller knows why the empty config returned.

Reply:
This is an issue I saw across all the three config files. I guess we need to make a separate issue for this and rectify this in one commit